.github/workflows/ci.yml

```diff
@@ -5,11 +5,21 @@
 # https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions
 
 name: ci
-on: [push, pull_request]
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
 jobs:
   smoke:
     runs-on: ubuntu-latest
     steps:
+      - name: Cancel previous
```

Comment:
Is it enough to add this only to the first job? Or we need this for all jobs?

There is also something about forks:
https://github.com/styfle/cancel-workflow-action#advanced-pull-requests-from-forks
but I don't fully understand that. Forks are not in google org, do they count against our limit?
Yes, we use forks.

And please call it `cancel previous`, we don't start jobs with capital letters.